src/zenml/cli/utils.py

```diff
@@ -1064,7 +1064,12 @@ def warn_unsupported_non_default_workspace() -> None:
 
 
 def print_page_info(page: Page[T]) -> None:
-    """Print all information pertaining to a page to show the amount of items and pages."""
+    """Print all information pertaining to a page to show the amount of items
+    and pages.
```

Comment:
Two liners in the beginning were also disallowed by pydocstyle I remember, so I guess neither darglint nor ruff cover this one?

Reply:
This was caught by the linting check. Fixed now, but Ruff is catching this.